### PR TITLE
Development

### PR DESCRIPTION
**Pull request by trox115**

### CODE REVIEW

**Comment by Torres-ssf** [comment]
### Congratulations @trox115  👍
You worked a lot in this project and should be proud of it. It's a huge project. But before submitting to master we need to do some change to make it match Microverse guidelines. They are:

- [x] Top Story, More Stories, Featured Stories are each one a separated section. Use 3 sections for each one of them.

- [x] Opinion container must be a `section` tag.

- [x] Each one of the 8 items in the opinion section must inside an `article` tag.

- [x] The Latest News section must be inside a `section` tag. All articles of it must be inside a `section` tag.

- [x]  The text from the latest article of the latest news section it's hidden in the lowers resolution. Please fix this.

- [x] The 'In the magazine' section must be inside a `<section>` tag.

- [x] The 'Editor's pick' must be inside a  `<section>` tag.

- [x] The 'Editor's pick' section heading must have a logo (like the original page).

- [x] The 'Featured Slideshow' must be inside a `section` tag. It seems that your 'Featured Slideshow' are using the same heading from the 'Editor's pick' section. The heading must include the 'Featured Slideshow' logo as well.

- Mobile \[Screen `width` up to `480px`\]

- [x] Try to make the logo, sign in button, and the subscribe buttons from the navbar a little smaller. This way they will behave better in this screen size.

- Medium \[Screen `width` from `768px` to `1024px`\]

- [x] For this resolution you have to match the same approach from the original page. The main content of the page must have only 2 columns instead of 3. Use 3 columns only in larger screen size. For this project, it's important to try to match the same appearance in each screen size and try to lay out the elements at the same way.

- Large \[Screen `width` from `1024px` and up\]

- [x] Use 3 columns for the main content only in this resolution. Again, it's important to try to match the style from the original page.

Please kindly make these changes and submit for code review again.

File: index.html [hunk 1]
```diff
@@ -0,0 +1,1914 @@
...
+      <div class="max-w">
+        <div class="date">
+          <p>Fri, Jun 21, 2019</p>
+        </div>
+        <div class="logo">
+          <img src="img/logo.png" alt="atributte" />
+        </div>
+        <div class="subscribe-top">
+          <a href="#">SIGN IN</a>
+          <a href="#" id="subscribe">Subscribe ></a>
```
Comment (bellom) [inline index.html:53]: kindly fix the background color for this, checkout the screenshot from your web page.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,1914 @@
...
+                
+                </div>
+              </div>
+            </article>
+            </section>
+              <!-- opinion -->
+              <section>
+              
+                <div class="title-block">
+                  <h2>Latest News</h2>
```
Comment (bellom) [inline index.html:443]: kindly work on this line, needs little more styling.

**Review by bellom — CHANGES_REQUESTED** [review]
Hi @trox115,
Great work so far !!
Just few things left, as previous reviewer said "it's important to try to match the style from the original page."
Kindly fix the following:
1. line 53 >> `subscribe` see the screenshot below;
2. line 443 >> `latest news` see the screenshot below;
Compare to the original site
image

Kindly fix those issue and submit another review, :+1:  await you....

**Review by bellom — APPROVED** [review]
Hi @trox115,
## Congratulation :+1: 
_What a great job !!_

**Pull request merged**